build-packages/check-public-api/check-public-api.spec.ts

```diff
@@ -59,64 +65,64 @@ describe('check-public-api', () => {
         }
       });
 
-      await expect(() =>
-        exportAllInBarrel('dir1', 'index.ts')
-      ).rejects.toThrowError("'index.ts' is not in sync.");
+      await exportAllInBarrel('dir1', 'index.ts');
 
       expect(errorSpy).toHaveBeenCalledWith(
         "'dir2' is not exported in 'dir1/index.ts'."
       );
+      expect(errorSpy).toHaveBeenCalledWith("'index.ts' is not in sync.");
     });
-  });
 
-  it('checkBarrelRecursive passes recursive check for barrel file exports', () => {
-    mock({
-      dir1: {
-        file1: '',
-        'index.ts': "export * from './file1'; export * from './dir2';",
-        dir2: {
-          file2: '',
-          file3: '',
-          'index.ts': "export * from './file2';export * from './file3';"
+    it('checkBarrelRecursive passes recursive check for barrel file exports', () => {
+      mock({
+        dir1: {
+          file1: '',
+          'index.ts': "export * from './file1'; export * from './dir2';",
+          dir2: {
+            file2: '',
+            file3: '',
+            'index.ts': "export * from './file2';export * from './file3';"
+          }
         }
-      }
+      });
+      checkBarrelRecursive('dir1');
     });
-    checkBarrelRecursive('dir1');
-  });
 
-  it('typeDescriptorPaths finds the .d.ts files and excludes index.d.ts', () => {
-    mock({
-      dir1: {
-        'file1.d.ts': '',
-        'index.d.ts': '',
-        dir2: {
-          'file2.d.ts': '',
-          'file3.d.ts': '',
-          'index.d.ts': ''
+    it('typeDescriptorPaths finds the .d.ts files and excludes index.d.ts', async () => {
+      mock({
+        dir1: {
+          'file1.d.ts': '',
+          'index.d.ts': '',
+          dir2: {
+            'file2.d.ts': '',
+            'file3.d.ts': '',
+            'index.d.ts': ''
+          }
         }
-      }
+      });
+
+      expect(await typeDescriptorPaths('dir1')).toEqual([
+        path.normalize('dir1/file1.d.ts'),
+        path.normalize('dir1/dir2/file3.d.ts'),
+        path.normalize('dir1/dir2/file2.d.ts')
+      ]);
     });
-    expect(typeDescriptorPaths('dir1')).toEqual([
-      'dir1/dir2/file2.d.ts',
-      'dir1/dir2/file3.d.ts',
-      'dir1/file1.d.ts'
-    ]);
   });
 
-  describe('parseTypeDefinitionFile', () => {
```

Comment:
[q] Not sure if i understood what changed here, did we rename the function?

Reply:
Yep. Just a name change. The current name makes more sense than what is was.